csharp_dotnetcore/12.NLP-With-LUIS/LuisBot/AppInsights/MyAppInsightLuisRecognizer.cs

```diff
@@ -0,0 +1,129 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
Per our discussion today, please remove app insights from this sample. This should be its own sample.

Reply:
I was going to do this in a different PR, but i'll add the mods into this one.